ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -898,3 +898,23 @@ if dali_support_is_available
     install_dir: nnstreamer_libdir
   )
 endif
+
+if llama2c_support_is_available
+  filter_sub_llama2c_sources = ['tensor_filter_llama2.cc']
+
+  nnstreamer_filter_llama2c_deps = [glib_dep, gst_dep, nnstreamer_dep, llama2c_support_deps]
```

Comment:
plz check dependency to filter sub-plugin,
```
[glib_dep, nnstreamer_single_dep, llama2c_support_deps]
```

Reply:
Remove gst_dep in llama2c dep
Thank you :)

Reply:
plz check dependency for nnst :)
`nnstreamer_dep -> nnstreamer_single_dep`

Reply:
Updated it! Sorry 😅